module/observations/helper.js

```diff
@@ -1772,21 +1773,41 @@ module.exports = class ObservationsHelper {
                 
                 let observationData = await this.observationDocuments({
                     _id : observationId,
-                },["_id","solutionId"]);
+                },["_id","solutionId","programId"]);
                 
                 let solutionData;
+                let programUsers;
+                let requestForPIIConsent;
                 if(observationData[0]){
 
                     solutionData = await solutionHelper.solutionDocuments({
                         "_id" : observationData[0].solutionId
 
                         },[
                             "allowMultipleAssessemts",
-                            "license"
+                            "license",
+                            "rootOrganisations"
```

Comment:
@VISHNUDAS-tunerlabs If we have rootOrganisations in solution in some other place I saw we are getting it explicitly from programs even though we have solutions